src/main/java/net/fabricmc/loom/build/nesting/NestedDependencyProvider.java

```diff
@@ -133,15 +129,16 @@ public static NestedDependencyProvider createNestedDependencyProviderFromConfigu
 				continue;
 			}
 
-			List<File> files = dependency
+			dependency
 					.getModuleArtifacts()
-					.stream()
-					.map(ResolvedArtifact::getFile)
-					.collect(Collectors.toList());
-
-			for (File file : files) {
-				fileList.add(new DependencyInfo<>(dependency, new ResolvedDependencyMetaExtractor(), file));
-			}
+					.forEach((artifact) -> fileList.add(
+							new DependencyInfo<>(
+									dependency,
+									new ResolvedDependencyMetaExtractor(),
+									artifact.getFile(),
+									artifact.getClassifier()
+							))
+					);
```

Comment:
I would recommend using a normal for-loop here.

Reply:
Yeah, that makes more sense. I used a stream since it previously did, but that's not necessary anymore.